src/browser/js/notification.js

```diff
@@ -8,14 +8,27 @@ import {throttle} from 'underscore';
 
 import {ipcRenderer, remote} from 'electron';
 
-import osVersion from '../../common/osVersion';
-import dingDataURL from '../../assets/ding.mp3'; // https://github.com/mattermost/platform/blob/v3.7.3/webapp/images/ding.mp3
+import bing from '../../assets/sounds/bing.mp3';
+import crackle from '../../assets/sounds/crackle.mp3';
+import down from '../../assets/sounds/down.mp3';
+import hello from '../../assets/sounds/hello.mp3';
+import ripple from '../../assets/sounds/ripple.mp3';
+import upstairs from '../../assets/sounds/upstairs.mp3';
+
+export const notificationSounds = new Map([
+  ['Bing', bing],
+  ['Crackle', crackle],
+  ['Down', down],
+  ['Hello', hello],
+  ['Ripple', ripple],
+  ['Upstairs', upstairs],
```

Comment:
should we also have the 'Ding' sound that is used in win7?

Reply:
If we are goint to support the old win7 sound (Ding) I think we should